.gitignore

```diff
@@ -0,0 +1,298 @@
+
```

Comment:
this is really exhausting gitignore for such a small thing - I would expect  just `.idea/` ... but whateever :shrug:

Reply:
I included everything I could think of, just to be sure that someone doesn't commit anything by accident